### ISSUE TITLE
Update documentation related to "nbdev_release"

### ISSUE THREAD

**Issue by bkowshik** [fastai/nbdev#1193]
Ref: https://nbdev.fast.ai/tutorials/tutorial.html#upload-to-pypi-and-conda

Trying to make a release of my package using `nbdev==2.3.7`. But, I don't see any command called `nbdev_release` which is mentioned in the documentation. Instead, I see the below similar looking options:

```bash
$ nbdev_release
zsh: command not found: nbdev_release


# On hitting TAB, I see the following options.
$ nbdev_release_
nbdev_release_both  nbdev_release_gh    nbdev_release_git
```

**Comment by seeM** [comment]
I believe it should refer to `nbdev_release_both`

**Comment by seeM** [comment]
Fixed by #1215.
